webpack/assets/javascripts/react_app/components/common/forms/__snapshots__/Actions.test.js.snap

```diff
@@ -83,21 +98,6 @@ exports[`actions should show a spinner when submitting 1`] = `
     >
        
       Submit
-      <span
-        className="fr"
-      >
-        <div
-          className="loader-root"
-        >
-          <Spinner
```

Comment:
is this a bug?

Reply:
The diff looks misleading because snapshots moved a bit as result of changes in tests, but the test snapshot does have a spinner when inspecting the snapshot file.